tensorflow-base-cpu-linwin/meta.yaml

```diff
@@ -39,23 +38,21 @@ requirements:
     - patch       # [unix]
     - m2-patch    # [win]
     # required here for all OS's, and ALSO below for windows
-    - bazel >=3.7.2
+    - bazel >=4.2.2,<5
     - aws-sdk-cpp
-    # - abseil-cpp
-    - cython 0.29.21  # [not aarch64]
-    - cython 0.29.*   # [aarch64]
+    - cython
```

Comment:
I'm not sure if we really need `cython` and `wheel` in req/build instead of `host`

Reply:
cython is a compiler tool, so should be in build/req. I'll drop the wheel from build/req. Thanks!